Products/product.js

```diff
@@ -0,0 +1,10 @@
+import hats from '../Data/data.js';
+import renderProducts from './render-products.js';
+
+const unorderedListElements = document.getElementById('hats');
+
+for (let i = 0; i < hats.length; i++) {
+    const hats = hats[i];
```

Comment:
const hats = hats[i] might be confusing for the computer since both variables are the same name. You will want to change of the variables. 
Maybe hat = hats[i]?